packages/@textlint/markdown-to-ast/test/fixtures/definition-newline.text/output.json

```diff
@@ -2,69 +2,11 @@
     "type": "Document",
     "children": [
         {
-            "type": "Paragraph",
-            "children": [
-                {
-                    "type": "LinkReference",
-                    "identifier": "baz",
-                    "referenceType": "shortcut",
-                    "children": [
-                        {
-                            "type": "Str",
-                            "value": "baz",
-                            "loc": {
-                                "start": {
-                                    "line": 1,
-                                    "column": 1
-                                },
-                                "end": {
-                                    "line": 1,
-                                    "column": 4
-                                }
-                            },
-                            "range": [
-                                1,
-                                4
-                            ],
-                            "raw": "baz"
-                        }
-                    ],
-                    "loc": {
-                        "start": {
-                            "line": 1,
-                            "column": 0
-                        },
-                        "end": {
-                            "line": 1,
-                            "column": 5
-                        }
-                    },
-                    "range": [
-                        0,
-                        5
-                    ],
-                    "raw": "[baz]"
-                },
-                {
-                    "type": "Str",
-                    "value": ": /url (\n)",
-                    "loc": {
-                        "start": {
-                            "line": 1,
-                            "column": 5
-                        },
-                        "end": {
-                            "line": 2,
-                            "column": 1
-                        }
-                    },
-                    "range": [
-                        5,
-                        15
-                    ],
-                    "raw": ": /url (\n)"
-                }
-            ],
+            "type": "Definition",
```

Comment:
- [ ] Definition changes a lot. need to check


> notable: references (as in, links [text][id] and images ![alt][id]) are no longer present as such in the syntax tree if they don’t have a corresponding definition ([id]: example.com). The reason for this is that CommonMark requires [text *emphasis start][undefined] emphasis end* to be emphasis.

https://github.com/remarkjs/remark/releases/tag/13.0.0